scripts/install/MapPasswordControlFeatureFlag.php

```diff
@@ -37,6 +40,16 @@ public function __invoke($params)
             SectionVisibilityFilter::OPTION_FEATURE_FLAG_SECTIONS,
             $featureFlagSections
         );
+        /** @var core_kernel_classes_Resource $ffResource */
+        $ffResource = $this->getServiceManager()
```

Comment:
You can solve this in SectionVisibility. As per doing something like this: 

In the config you will have: 

```
return new oat\tao\model\menu\SectionVisibilityFilter(array(
    // Current behavior
    'featureFlagSections' => array(
        'section_to_show_1' => array(
            'FEATURE_FLAG_SHOW_EXAMPLE1'
        ),
        'section_to_show_2' => array(
            'FEATURE_FLAG_SHOW_EXAMPLE2'
        ),
    ),
    // New behavior
    'featureFlagHideSections' => array(
        'section_to_hide_1' => array(
            'FEATURE_FLAG_HIDE_EXAMPLE1'
        ),
        'section_to_hide_2' => array(
            'FEATURE_FLAG_HIDE_EXAMPLE2'
        ),
    )
));
```

Add the calss will be something like: 

```
class SectionVisibilityFilter extends ConfigurableService implements SectionVisibilityFilterInterface
{
    public const SERVICE_ID = 'tao/SectionVisibilityFilter';
    public const OPTION_FEATURE_FLAG_SECTIONS = 'featureFlagSections';
    public const OPTION_FEATURE_FLAG_HIDE_SECTIONS = 'featureFlagHideSections';

    /**
     * @throws LogicException
     */
    public function isVisible(string $section): bool
    {
        $sections = $this->getOption(self::OPTION_FEATURE_FLAG_SECTIONS);
        $sectionsHide = $this->getOption(self::OPTION_FEATURE_FLAG_HIDE_SECTIONS);

        if (empty($sections[$section])) {
            return true;
        }

        foreach ($sectionsHide[$section] as $featureFlag) {
            if ($this->getFeatureFlagChecker()->isEnabled($featureFlag)) {
                return false;
            }
        }

        foreach ($sections[$section] as $featureFlag) {
            if (!$this->getFeatureFlagChecker()->isEnabled($featureFlag)) {
                return false;
            }
        }

        return true;
    }

    private function getFeatureFlagChecker(): FeatureFlagCheckerInterface
    {
        return $this->getServiceLocator()->get(FeatureFlagChecker::class);
    }
}
```

Reply:
If you are applying this change, please update the doc here https://github.com/oat-sa/tao-core/tree/master/models/classes/featureFlag#sectionvisibilityfilterinterface